traffic_ops/install/bin/postinstall

```diff
@@ -48,6 +49,10 @@ my $tmAdminUser = "";
 my $tmAdminPw = "";
 my $parameters;
 
+my $skip_ssl = 0;
+GetOptions("skip-ssl" => \$skip_ssl)
```

Comment:
how about adding a --help?   No way to tell what the option is without looking thru the source..